.travis.yml

```diff
@@ -15,4 +15,8 @@ before_script:
 script:
 - export NODE_PATH=$TRAVIS_BUILD_DIR'/privly.safariextension/privly-applications/test/node_modules'
 - cd test
-- ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js
+- "if [ ${TRAVIS_PULL_REQUEST} = 'false' ]; then ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js; fi" #Run the unit tests
+- "cd ../privly.safariextension/privly-applications/test"
+- "if [ ${TRAVIS_PULL_REQUEST} = 'false' ]; then ./node_modules/.bin/lcov-result-merger 'coverage/*.lcov' 'coverage/combined.lcov'; fi" # Merge the coverage results
```

Comment:
You should not need to use the lcov result merge. Write all your JS so that it can sit in the same namespace, then you will only need to run the karma tests with a single command.